components/autofill/src/sync/address/incoming.rs

```diff
@@ -199,7 +199,7 @@ impl ProcessIncomingRecordImpl for IncomingAddressesImpl {
         });
 
         match result {
-            Ok(r) => Ok(Some((incoming.guid.clone(), r))),
```

Comment:
This is actually where the bug was - it should be the local record ID - but we are returning that local record, so I thought I'd simplify and just drop the GUID entirely as it made the error less obvious.